eth/rewardservice.go

```diff
@@ -91,30 +91,11 @@ func (s *RewardService) tryReward() error {
 			return err
 		}
 
-		err = s.client.CheckTx(tx)
-		if err != nil {
-			if err == context.DeadlineExceeded {
-				glog.Infof("Reward tx did not confirm within defined time window - will try to replace pending tx once")
-				// Previous attempt to call reward() still pending
-				// Replace pending tx by bumping gas price
-				tx, err = s.client.ReplaceTransaction(tx, "reward", nil)
-				if err != nil {
-					return err
-				}
-				if err := s.client.CheckTx(tx); err != nil {
-					return err
-				}
-			} else {
-				return err
-			}
-		}
-
-		tp, err := s.client.GetTranscoderEarningsPoolForRound(s.client.Account().Address, currentRound)
-		if err != nil {
+		if err := s.client.CheckTx(tx); err != nil {
 			return err
 		}
 
-		glog.Infof("Called reward for round %v - %v rewards minted", currentRound, FormatUnits(tp.RewardPool, "LPTU"))
+		glog.Infof("Called reward for round %v", currentRound)
```

Comment:
Just noting that this is related to https://github.com/livepeer/go-livepeer/issues/1822

Reply:
Yeah sorry for including it here, it doesn't fully address #1822 in that rather than fixing the value shown I decided to remove it for now to avoid confusion.

Reply:
Yep that's fine I was just making this note for myself as a reminder. I'd actually even be fine with closing that issue as the main problem was incorrect/confusing information was being logged and if there is need to log the amount minted a separate issue can track that.